src/main/java/stream/flarebot/flarebot/permissions/PerGuildPermissions.java

```diff
@@ -93,4 +86,19 @@ public boolean isContributor(net.dv8tion.jda.core.entities.User user) {
         return user.getId().equals("215644829969809421");
     }
 
+    public void createDefaultGroup() {
+        if (hasGroup("Default")) {
+            deleteGroup("Default");
+        }
+        Group defaults = new Group("Default");
+        for (Command command : FlareBot.getInstance().getCommands()) {
+            if (command.isDefaultPermission()) {
+                defaults.addPermission(command.getPermission());
+            }
+        }
+        defaults.addPermission("flarebot.userinfo.other");
+        defaults.addPermission("flarebot.playlist.clear");
```

Comment:
we should find a better way to handle this

Reply:
Yup, not for this update tho

Reply:
I'm fine with that